loader-tests/check-feeds.ts

```diff
@@ -0,0 +1,36 @@
+import yaml from 'yaml'
+
+import {
+  enableTestClient,
+  error,
+  type Feed,
```

Comment:
Don’t like that we have an export (not an internal type as before) with name conflict with `core`’s `Feed`.

Can we rename it?